Makefile

```diff
@@ -21,13 +28,15 @@ ifdef E2E_TEST_EXTRA_ARGS
 	E2E_TEST_ARGS +=  $(E2E_TEST_EXTRA_ARGS)
 endif
 
+GINKGO?= build/_output/bin/ginkgo
```

Comment:
`GINKGO ?=`

Reply:
Yep it will include a space " += " has one, I will fix the GINKGO part though